src/ocamlorg/lib/package.mli

```diff
@@ -46,7 +46,7 @@ module Info : sig
     ; homepage : string list
     ; tags : string list
     ; dependencies : (Name.t * string option) list
-    ; rev_deps : (Name.t * string option) list
+    ; rev_deps : (Name.t * string option * OpamPackage.t) list
```

Comment:
I would just store the `Version.t` as an `OpamPackage.t` is a name and version but we already know the name from the first element of the triple.

Reply:
Oh, I should have taken more time to notice that. Sorry.

Reply:
After I changed it to `Version.t`, I got this error:

![i3 p1](https://user-images.githubusercontent.com/78751231/142659881-e4bc63a6-8f82-4f67-a39e-fc3085498336.png)

Reply:
Yep the types in `ml` and `mli` need to be the same :)) There's some explanation of `ml` and `mli` here: https://dev.realworldocaml.org/files-modules-and-programs.html#scrollNav-3

Reply:
Thanks for the reference :) So that means we're taking back to `OpamPackage.t` ?

Reply:
Not quite, it means updating the implementation to return just the version and not create the package. Concretely changing the type to match the `mli` https://github.com/JiaeK/v3.ocaml.org-server/blob/217f4a8173314f7cd307504d138d7080b42a830a/src/ocamlorg/lib/package.ml#L26 and update the `latest` calculation to only return the latest version https://github.com/JiaeK/v3.ocaml.org-server/blob/217f4a8173314f7cd307504d138d7080b42a830a/src/ocamlorg/lib/package.ml#L139